Automattic/woocommerce-payments
# Update customer multi-currency default settings and options

Fixes #2091 

#### Changes proposed in this Pull Request

- Update the "Charm pricing" label to read "Price charm"
- Remove the tooltips for Price rounding and charm
- Remove the "Learn more" links from the Price rounding and charm descriptions
- Remove the "(recommended)" label from the -0.01 charm option
- Fix typo in preview description
- Modify get_ceil_price to accept an arbitrary rounding value
- Update decimal currency rounding options
- Add zero decimal currencies price rounding and charm options
- Update default price rounding option

It's worth noting that as part of this change, the `get_ceil_price` logic has been altered to work with arbitrary values, such as `0.25` and `0.5`. This is a breaking change with the previous values used by the rounding options, but it was not addressed automatically as part of this PR because multi-currency is not being used by merchants yet.









#### Testing instructions





* Make sure all the changes requested in #2091 are implemented by this PR
* Check that the new price rounding and charm options work properly in the shop and preview
* Check that the appropriate options for zero-decimal currencies are displayed

-------------------
